src/App.jsx

```diff
@@ -14,70 +15,71 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export default function App() {
+  const jam = goods.find(e => e === 'Jam');
+  const [item, setItem] = useState(jam);
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  function itemPick(elem) {
+    setItem(elem);
+  }
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  function itemClear() {
+    setItem('');
+  }
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+  return (
+    <main className="section container">
+      {item === '' ? (
+        <h1 className="title is-flex is-align-items-center">
+          No goods selected
+        </h1>
+      ) : (
+        <h1 className="title is-flex is-align-items-center">
+          {`${item} is selected`}
+          <button
+            onClick={() => itemClear()}
```

Comment:
you can write this handler like `onClick={itemClear}`

check in other places where you used this handler as well